Using device: cuda
Traceback (most recent call last):
  File "/net/tscratch/people/plgmpniak/KITS_project/Kits_23/train/train.py", line 77, in <module>
    main()
  File "/net/tscratch/people/plgmpniak/KITS_project/Kits_23/train/train.py", line 28, in main
    train_loader = DataLoader(train_set, batch_size=2, shuffle=True, num_workers=4, pin_memory=True)
  File "/net/tscratch/people/plgmpniak/miniconda3/envs/kits23-unet/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 376, in __init__
    sampler = RandomSampler(dataset, generator=generator)  # type: ignore[arg-type]
  File "/net/tscratch/people/plgmpniak/miniconda3/envs/kits23-unet/lib/python3.10/site-packages/torch/utils/data/sampler.py", line 164, in __init__
    raise ValueError(
ValueError: num_samples should be a positive integer value, but got num_samples=0
